keel-sql/src/test/resources/application.properties

```diff
@@ -11,6 +11,6 @@ spinnaker.baseUrl=http://spinnaker
 services.fiat.baseUrl=https://fiat.net
 spring.application.name=keel
 sql.enabled=true
-sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
-sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
+sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/keel&TC_REUSABLE=true
+sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/keel&TC_REUSABLE=true
```

Comment:
That `TC_REUSABLE` flag is how you tell testcontainers to re-use the container via the connection string